project-reports/2024/2024-annual-Hyperledger-Cacti.md

```diff
@@ -0,0 +1,63 @@
+---
+layout: default
+title: 2024 Annual Review Hyperledger Cacti
+parent: 2024
+grand_parent: Project Updates
+---
+
+# Project Health
+
+[Hyperledger Cacti LFX Insights page](https://insights-v2.lfx.linuxfoundation.org/cactus/trends)
+
```

Comment:
It is not clear whether annual reports should focus on the 'old' Insights page as posted here, or the 'new' Insights page which for Cacti would be:
https://insights.lfx.linuxfoundation.org/foundation/hyp/overview?project=cactus&repository=all&dateFilters=Last%20Year&dateRange=2023-01-01%20to%202023-12-31&compare=PP&granularity=month&hideBots=true

In the Fabric report I focused on the 'new' Insights page, maybe both are important?

Reply:
On the new Insights page for Cacti:

https://insights.lfx.linuxfoundation.org/foundation/hyp/overview?project=cactus&repository=all&dateFilters=Last%20Year&dateRange=2023-01-01%20to%202023-12-31&compare=PP&granularity=month&hideBots=true

Organization Dependency widget:

![image](https://github.com/hyperledger/toc/assets/20165681/19f31efa-428a-4b02-8c90-f13ca4830c93)

Reply:
It looks like Insights has been updated, the primary org has been updated from "Hyperledger Labs" to "Accenture". I'd still recommend including some of the 'new' Insights dashboard 2023 metrics in the annual report, e.g.:
- 80 contributors (+21% from 2022)
- 497 commits (-52% from 2022)
- 51% of code activities were performed by one person (from Accenture)
- 60% of code activities were performed by 1 organization (Accenture)

Reply:
@denyeart Thank you very much and sorry for the slow updates! I've swapped out the link just now to the new one!